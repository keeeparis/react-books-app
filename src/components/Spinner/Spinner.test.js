import renderer from 'react-test-renderer'
import Spinner from './Spinner'

it('render spinner', () => {
  const component = renderer.create(<Spinner />).toJSON()
  expect(component).toMatchSnapshot()
})
