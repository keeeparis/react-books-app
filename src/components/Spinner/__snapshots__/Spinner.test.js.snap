// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render spinner 1`] = `
<div
  className="ldsRoller"
>
  <div />
  <div />
  <div />
  <div />
  <div />
  <div />
  <div />
  <div />
</div>
`;
